ios/Pods/Manifest.lock

```diff
@@ -1,15 +1,56 @@
 PODS:
   - Flutter (1.0.0)
+  - flutter_smkit_ui (0.1.0):
```

Comment:
@SencyNetanel Are this items needed to be tracked? or can we add it to .gitignore?

Reply:
those items needed to be tracked